From 9601a6df09f881583c441d1a308878b360432dc5 Mon Sep 17 00:00:00 2001
From: Dom Cobley <popcornmix@gmail.com>
Date: Fri, 12 Mar 2021 11:26:29 +0000
Subject: [PATCH 0585/2201] vc4/drm: Increase hdmi audio axi priority to avoid
 lost samples

With HBR audio (8 channel 192kHz) we get occasional VC4_HD_MAI_CTL_DLATE error flags in
HDMI_MAI_CTL which seem to correspond with audio dropouts.

Increasing the normal AXI priority for dma is needed to avoid these

Signed-off-by: Dom Cobley <popcornmix@gmail.com>
---
 arch/arm/boot/dts/bcm2711-rpi.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2711-rpi.dtsi b/arch/arm/boot/dts/bcm2711-rpi.dtsi
index 8053ef54acc8..20581ee97827 100644
--- a/arch/arm/boot/dts/bcm2711-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2711-rpi.dtsi
@@ -179,7 +179,7 @@
 };
 
 &hdmi0 {
-	dmas = <&dma (10|(1<<27)|(1<<24)|(0<<16)|(15<<20))>;
+	dmas = <&dma (10|(1<<27)|(1<<24)|(10<<16)|(15<<20))>;
 	status = "disabled";
 };
 
@@ -188,7 +188,7 @@
 };
 
 &hdmi1 {
-	dmas = <&dma (17|(1<<27)|(1<<24)|(0<<16)|(15<<20))>;
+	dmas = <&dma (17|(1<<27)|(1<<24)|(10<<16)|(15<<20))>;
 	status = "disabled";
 };
 
-- 
2.17.1

